nss/netbsdnss.c

```diff
@@ -0,0 +1,542 @@
+/*
+ * This file is derived from NetBSD stub for PADL nss_ldap
+ * http://cvsweb.netbsd.org/bsdweb.cgi/pkgsrc/databases/nss_ldap/files/netbsd.c 
```

Comment:
Can you provide some more information on the copyright holders and license of the original file?